src/main/resources/application.yaml

```diff
@@ -187,6 +187,9 @@ darts:
   dar-pc-notification:
     url-format: http://{0}/VIQDARNotifyEvent/DARNotifyEvent.asmx
   events:
+    duplicates:
+      earliest-removable-event-date: ${MODERNISED_DARTS_START_DATE:2024-07-16}
```

Comment:
Have you got a flux change to set this per environment? On prod, we want to make sure this is clear it needs to be set to the go live date, so probably needs a comment explaining.

Reply:
https://github.com/hmcts/sds-flux-config/pull/5139